src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotificationFactory.java

```diff
@@ -232,27 +296,33 @@
             List<String> archiveErrors) {
         String title;
         String text;
+        String[] placeholderValues;
         switch (notificationType) {
             case COURSE_ARCHIVE_STARTED -> {
                 title = COURSE_ARCHIVE_STARTED_TITLE;
-                text = "The course \"" + course.getTitle() + "\" is being archived.";
+                text = COURSE_ARCHIVE_STARTED_TEXT;
+                placeholderValues = new String[] { course.getTitle() };
             }
             case COURSE_ARCHIVE_FINISHED -> {
                 title = COURSE_ARCHIVE_FINISHED_TITLE;
-                text = "The course \"" + course.getTitle() + "\" has been archived.";
-
-                if (!archiveErrors.isEmpty()) {
-                    text += " Some exercises couldn't be included in the archive:<br/><br/>" + String.join("<br/><br/>", archiveErrors);
+                if (archiveErrors.isEmpty()) {
+                    text = COURSE_ARCHIVE_FINISHED_WITHOUT_ERRORS_TEXT;
```

Comment:
I do not think we want to apply formatting (like html tags) to placeholder values. This will become very hard to handle on the native client side, where I basically have to reverse this process to extract each archive error. This should be part of the UI. I think you can just add all archiveErrors into the array.

Reply:
I removed them. Instead, I listed the errors separated by commas.